src/Sylius/Bundle/OrderBundle/Doctrine/ORM/OrderRepository.php

```diff
@@ -56,6 +56,22 @@ public function findLatest($count)
     /**
      * {@inheritdoc}
      */
+    public function createCartsPaginator(array $criteria = [], array $sorting = [])
```

Comment:
Not sure I understand what do you mean - By "grids" I understand grid configuration, not routing generation.